src/main/java/frc/robot/commands/rgb/rainbow.java

```diff
@@ -2,22 +2,24 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-import frc.robot.subsystems.rgb;
+import frc.robot.subsystems.RGB;
 
-public class rainbow extends CommandBase{
-    private final rgb rgb;
-    public rainbow(rgb subsystem) {
+public class rainbow extends CommandBase {
+    private final RGB rgb;
+
+    public rainbow(RGB subsystem) {
         addRequirements(subsystem);
         rgb = subsystem;
     }
 
     @Override
     public void execute() {
         rgb.setRainbow();
-        
+
     }
 
     @Override
     public void end(boolean interrupted) {
+
     }
-} 
+}
```

Comment:
Generally we need a new line at the end of our code. As you can see Github even highlight it here (the red circle with a minus sign inside). So just add a new line at the end.

Reply:
not in issue

Reply:
Oops, I missed. Ok then!